lib/cisco_os_shim/nxapi/client.rb

```diff
@@ -18,8 +18,6 @@
 
 require 'json'
 require 'net/http'
-require_relative '../cisco_logger'
-require_relative '../client'
```

Comment:
As noted this seems to have been a leftover from initial work I had done - will re-add.